src/world/area_pra/pra_35/D8D270.c

```diff
@@ -38,8 +38,100 @@ INCLUDE_ASM(s32, "world/area_pra/pra_35/D8D270", func_80241AB4_D8ED04);
 
 #include "world/common/UnkFunc7.inc.c"
 
-//#include "world/common/UnkNpcAIFunc27.inc.c"
-
+// Requires data migration
+#ifdef NON_MATCHING
+ApiStatus func_802423CC_D8F61C(Evt *script, s32 isInitialCall) {
+    Enemy* enemy = script->owner1.enemy;
+    Npc *npc = get_npc_unsafe(enemy->npcID);
+    Bytecode* args = script->ptrReadPos;
+    EnemyTerritoryThing territory;
+    EnemyTerritoryThing* territoryPtr = &territory;
+    NpcAISettings* npcAISettings = (NpcAISettings*)evt_get_variable(script, *args++);
+
+    territory.unk_00 = 0;
+    territory.shape = enemy->territory->patrol.detectShape;
+    territory.pointX = enemy->territory->patrol.detect.x;
+    territory.pointZ = enemy->territory->patrol.detect.z;
+    territory.sizeX = enemy->territory->patrol.detectSizeX;
+    territory.sizeZ = enemy->territory->patrol.detectSizeZ;
+    territory.unk_18 = 65.0f;
+    territory.unk_1C = 0;
+
+    if (isInitialCall || (enemy->unk_B0 & 4)) {
+        script->functionTemp[0] = 0;
+        npc->duration = 0;
+        npc->currentAnim.w = enemy->animList[0];
+        npc->flags &= ~0x800;
+        if (!enemy->territory->patrol.isFlying) {
+            npc->flags = (npc->flags | 0x200) & ~0x8;
```

Comment:
these should be broken into separate statements (|= 0x200, &= ~0x8) (and below too)